go.mod

```diff
@@ -7,6 +7,8 @@ go 1.24.0
 // TODO: remove this entry after https://github.com/googleapis/google-cloud-go/issues/11448 is fixed
 replace cloud.google.com/go/storage => cloud.google.com/go/storage v1.43.0
 
+replace github.com/valyala/gozstd => github.com/AndrewChubatiuk/gozstd v0.0.0-20250301095332-4c0a10558f61
```

Comment:
It's fine for testing. But this change cannot be merged. We have to add changes into gozstd upstream at first.

Reply:
just wanted to know your opinion if it's worth waiting till it's merged or should we just create custom zstd reader, which embeds reader from gozstd and implements zstd.Reader interface

Reply:
It is better to re-use already existing zstd reader from the `cluster` branch, which is used for reading zstd-compressed data sent by `vminsert` into `vmstorage`. See https://github.com/VictoriaMetrics/VictoriaMetrics/blob/6a48fd49dec82fd1030769f3ee0f7450ecf71b86/lib/encoding/zstd/stream_pure.go#L13 and https://github.com/VictoriaMetrics/VictoriaMetrics/blob/6a48fd49dec82fd1030769f3ee0f7450ecf71b86/lib/encoding/zstd/stream_cgo.go#L12

Reply:
done